tx_check.go

```diff
@@ -85,7 +85,7 @@ func (tx *Tx) checkBucket(b *Bucket, reachable map[common.Pgid]*common.Page, fre
 		}
 
 		// Ensure each page is only referenced once.
-		for i := common.Pgid(0); i <= common.Pgid(p.Overflow()); i++ {
+		for i := common.Pgid(p.Id()); i <= common.Pgid(p.Overflow()); i++ {
 			var id = p.Id() + i
```

Comment:
I don't think so. `id` has `p.Id()` as base

Reply:
yes exactly, the base should be `Pgid(p.Id())` 

currently, it is `Pgid(0)`  regardless of the rootPageId we starting from this specific func call 

please correct me if i misunderstanding :)

Reply:
The change isn't correct. Let's work with an example, 

Assuming` p.Id() == 100`, and `overflow == 3`, then we need to iterate pages: 100, 101, 102.

Reply:
ahhhh i got it now, `p.Overflow()` contains count, not the pageIds 👍🏽 

sorry my bad 👍🏽